### PR TITLE
fix: added support for more complex statement decleration

### PR DESCRIPTION
**Pull request by samthakur587**

### CODE REVIEW

File: crosstl/src/translator/codegen/directx_codegen.py [hunk 1]
```diff
@@ -50,6 +49,7 @@ def __init__(self):
         }
 
     def generate(self, ast):
+        print(ast)
```
Comment (NripeshN) [inline crosstl/src/translator/codegen/directx_codegen.py:52]: @samthakur587 
Is there a reason why we are adding this print statement?

**Review by NripeshN — CHANGES_REQUESTED** [review]

File: crosstl/src/translator/codegen/directx_codegen.py (hunk 1 above)
Comment (samthakur587) [reply crosstl/src/translator/codegen/directx_codegen.py]: it's for debugging i removed it

**Comment by vaatsalya123** [comment]
Lgtm! Awesome work Sam.

**Review by vaatsalya123 — APPROVED** [review]

**Review by NripeshN — APPROVED** [review]
LGTM!

**Pull request merged**